arangod/Agency/AsyncAgencyComm.cpp

```diff
@@ -209,9 +209,10 @@ arangodb::AsyncAgencyComm::FutureResult agencyAsyncInquiry(AsyncAgencyCommManage
                     result.response().header.metaByKey(arangodb::StaticStrings::Location);
                 redirectOrError(man, endpoint, location);
                 return ::agencyAsyncInquiry(man, std::move(meta), std::move(body));
+              } else if (result.statusCode() != fuerte::StatusServiceUnavailable) {
```

Comment:
Please add a comment to explain the rationale.

Reply:
done